pkgs/by-name/ca/canon-capt/package.nix

```diff
@@ -0,0 +1,68 @@
+{
+  autoconf,
+  automake,
+  cups,
+  fetchurl,
+  stdenv,
+  unzip,
+}:
+let
+  version = "0.1.4.2-GxB";
+in
+stdenv.mkDerivation {
+  pname = "canon-capt";
+  inherit version;
+
+  src = fetchurl {
+    url = "https://github.com/mounaiban/captdriver/archive/8ecc3cde1ae9a20dcb015994bb0fea0dbefa2b83.zip";
+    sha256 = "sha256-QE9GiRC5ExH+LlooJ2eEFtsmA1aWzRoRfxh92EX6Tb8=";
+  };
```

Comment:
You should use fetchFromGitHub - if you can't, maybe add a comment on why it won't work for this package.